source/MaterialXGenMdl/Nodes/SourceCodeNodeMdl.cpp

```diff
@@ -36,6 +73,19 @@ void SourceCodeNodeMdl::initialize(const InterfaceElement& element, GenContext&
         {
             size_t pos = _functionSource.find_first_of('(');
             string functionName = _functionSource.substr(0, pos);
+
+            const ShaderGenerator& shadergen = context.getShaderGenerator();
+            const MdlShaderGenerator& shadergenMdl = static_cast<const MdlShaderGenerator&>(shadergen);
+            StringVec code = replaceSourceCodeMarkers(getName(), functionName,
+                [&shadergenMdl, &context](const string& marker)
+                {
+                    if (marker == MARKER_MDL_VERSION_SUFFIX)
+                    {
+                        return shadergenMdl.getMdlVersionFilenameSuffix(context);
```

Comment:
It looks like the result of getMdlVersionFilenameSuffix is constant. Maybe the call could be done outside here and the result captured to the lambda, instead of calling the function every time?